Feature/qu 1948

bkyrlach opened the pull request.

sherfert commented on ast/src/main/scala/org/opencypher/v9_0/ast/Clause.scala:
```diff
@@ -171,7 +171,7 @@ trait SingleRelTypeCheck {
 final case class ConstructGraph(
                                  clones: List[Clone] = List.empty,
-                                 news: List[New] = List.empty,
+                                 news: List[CreateInConstruct] = List.empty,
```
`news` here should probably be renamed as well.

sherfert commented on parser/src/main/scala/org/opencypher/v9_0/parser/Clauses.scala:
```diff
@@ -55,8 +55,8 @@ trait Clauses extends Parser
 
-  def New: Rule1[ast.New] = rule("NEW (construct subclause)") {
-    group(keyword("NEW") ~~ Pattern) ~~>> (ast.New(_))
+  def New: Rule1[ast.CreateInConstruct] = rule("NEW (construct subclause)") {
```
This rule should be renamed as well.

LeifW committed:
ScalaTest is a test dependency, not a runtime one.

This had mean anything that depended on "expressions" was also pulling
in ScalaTest (and its dependencies) as a compile/runtime dependency.

bkyrlach committed:
Merge pull request #1 from thatdot/scope_scalatest_to_test

ScalaTest is a test dependency, not a runtime one.

bkyrlach committed:
Updated build to publish to S3 bucket.
Changed group-id to avoid confusion between OpenCypher releases and our custom 2.13 build.
Incremented version number and removed `SNAPSHOT` tag

bkyrlach committed:
Updated version to separate our build from the official OpenCypher release.

bkyrlach committed:
New CALL RECURSIVELY syntax for recursive execution of a Cypher query as part of a larger query.
